### PR TITLE
BE: LOK : 2674 : HTTP/HTTPS service configuration & monitoring.

### PR DESCRIPTION
**Pull request by Junaid-Khan-Nant**
## Description
Ensure http/https service can be configured and the service is successfully monitored.

## Jira link(s)
https://opennms.atlassian.net/browse/LOK-2674.

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: minion/plugins/ntp-plugin/src/main/java/org/opennms/lokahi/minion/ntp/NtpMonitor.java [hunk 1]
```diff
@@ -129,7 +129,8 @@ private double pollNtpServer(DatagramSocket socket, InetAddress ipAddr, NTPMonit
     private void sendNtpRequest(DatagramSocket socket, InetAddress ipAddr, NTPMonitorRequest request)
             throws IOException {
         byte[] data = new NtpMessage().toByteArray();
-        DatagramPacket outgoing = createDatagramPacket(data, ipAddr, request.getPort(0));
+        DatagramPacket outgoing =
+                createDatagramPacket(data, ipAddr, request.getListPorts().getPort(0));
```
Comment (cgorantla) [inline minion/plugins/ntp-plugin/src/main/java/org/opennms/lokahi/minion/ntp/NtpMonitor.java:133]: Is there a check to validate whether at least one port exists ?
swhat about other ports, do we ever use them ?

File: shared-lib/inventory/src/main/proto/simpleMonitoredEntity.proto [hunk 2]
```diff
@@ -58,8 +59,13 @@ message SimpleMonitoredEntityRequest {
...
 
+message SimpleMonitorEntityStateList {
+  repeated MonitoredEntityStateDTO entry = 1;
+}
+
 service SimpleMonitoredEntityService {
   rpc list(google.protobuf.Empty) returns(SimpleMonitoredEntityResponseList) {};
   rpc upsert(SimpleMonitoredEntityRequest) returns(SimpleMonitoredEntityResponse) {};
   rpc delete(google.protobuf.StringValue) returns(google.protobuf.BoolValue) {};
+  rpc listAllSimpleMonitors(google.protobuf.Empty) returns(SimpleMonitorEntityStateList){};
```
Comment (cgorantla) [inline shared-lib/inventory/src/main/proto/simpleMonitoredEntity.proto:70]: Shouldn't this be moved to MonitoredEntityStatusService in montioredServiceStatus.proto ?

**Review by cgorantla — COMMENTED** [review]
few comments otherwise LGTM!

File: shared-lib/inventory/src/main/proto/simpleMonitoredEntity.proto (hunk 2 above)
Comment (Junaid-Khan-Nant) [reply shared-lib/inventory/src/main/proto/simpleMonitoredEntity.proto]: Done.

File: minion/plugins/ntp-plugin/src/main/java/org/opennms/lokahi/minion/ntp/NtpMonitor.java (hunk 1 above)
Comment (Junaid-Khan-Nant) [reply minion/plugins/ntp-plugin/src/main/java/org/opennms/lokahi/minion/ntp/NtpMonitor.java]: As verified from in classic we don't get list of ports from there. So i revert it back and go with single port.

**Review by cgorantla — APPROVED** [review]
LGTM!

**Pull request merged**